### PR TITLE
Create-go-skeleton

### LINKED ISSUE #1: Create go skeleton
To initilize your project structure we have to setup a skeleton strucuture with simple content e.g. simple files in it like simple functions with 1 +2 ...
Furtheremore we want to create our first test file.

**AC:**
* skeleton for go code is set up.
* First test is created.

### PR DESCRIPTION
**Pull request by LaurinStreng**
Close #1 
## What happened
Adds the following:
- [x] Skeleton
- [x] Github Actions (Linting and Testing)
- [x] Go Files with simple math and testing for this 

## How to review
- The Skeleton is fitting for our project
- Github Actions is Working
- Codereview 
- execute `go run ./cmd/api` you shoud get the number two

### CODE REVIEW

File: cmd/api/main.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	calc "ecmanager/internal/api"
+)
+
+func main() {
+	calc.Calculate()
```
[the remark this answers is not in the document]
Comment (LaurinStreng) [reply cmd/api/main.go:8]: Yes it work you can test it with: `go run ./cmd/api` in the project folder

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Binaries for programs and plugins
```
[the remark this answers is not in the document]
Comment (LaurinStreng) [reply .gitignore:1]: Done

File: .github/workflows/go-test.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: Go-test
```
[the remark this answers is not in the document]
Comment (LaurinStreng) [reply .github/workflows/go-test.yml:1]: Linter is added

File: cmd/api/main.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (tzabbi) [reply cmd/api/main.go:8]: ahh nice, new learning for me :)

**Review by tzabbi — APPROVED** [review]
Great work @LaurinStreng <3

**Review by Marcel2603 — APPROVED** [review]

**Pull request merged**